apispec/ext/marshmallow/swagger.py

```diff
@@ -231,7 +231,7 @@ def field2property(field, spec=None, use_refs=True, dump=True, name=None):
         ret['format'] = fmt
 
     default = field.default if dump else field.missing
-    if default:
+    if not isinstance(default, _Missing):
```

Comment:
Since `_Missing` is private API, you can compare against the `missing` singleton instead:

```python
from marshmallow import missing

# ...

if default is not missing:
```